docs/blog/0.10.md

```diff
@@ -0,0 +1,25 @@
+---
+date: 2022-09-15
+image: https://user-images.githubusercontent.com/5587788/190299248-7bc2e6f2-2aee-4415-aa38-dda784f75f6f.png
+
+---
+
+# Rill 0.10 – Expanded leaderboard metrics and nightly binaries
+
+:::note
+⚡ Rill Developer is a tool that makes it effortless to transform your datasets with SQL and create powerful, opinionated dashboards. These are our release notes for our `0.10` release, still in Tech Preview.
+
+To [try out Rill Developer, check out these instructions](/#pick-an-install-option) and [let us know over on Discord](https://bit.ly/3bbcSl9) if you encounter any problems or have ideas about how to improve Rill Developer!
```

Comment:
```suggestion
To [try out Rill Developer, check out these instructions](/#pick-an-install-option) and [let us know over on Discord](https://rilldata.link/cli) if you encounter any problems or have ideas about how to improve Rill Developer!
```
@magorlick we have a new shortened link

Reply:
This is not a user-facing link and we are going to keep the current version to help us understand CTR for different community engagement touch points